### PR TITLE
Add initial draft to include new FMI 3.0 data types

### PR DESCRIPTION
**Pull request by pmai**
This currently handles all new data types except for clocks. It does not
handle arrays.

### CODE REVIEW

**Review by mdeppe — APPROVED** [review]

**Comment by mdeppe** [comment]
Looks, nice. Approved by me.

File: schema/SystemStructureCommon.xsd [hunk 1]
```diff
@@ -409,23 +488,23 @@
                     <xs:documentation xml:lang="en">
                         This element provides for a transformation of integer parameter values
                         based on a mapping table and is valid for parameters of integer and 
-                        enumeration type.  Each mapping table entry is provided by a MapEntry
+                        enumeration types.  Each mapping table entry is provided by a MapEntry
```
Comment (klausschuch) [inline schema/SystemStructureCommon.xsd:491]: nitpick: In the spec we use the snippet "of all the integer types and Enumeration type" and here "of integer and enumeration types". I think we should decide which formulation to use and use the same for both places.

Comment (klausschuch) [inline schema/SystemStructureCommon.xsd:491]: nitpick: In the spec we use the snippet "of all the integer types and Enumeration type" and here "of integer and enumeration types". I think we should decide which formulation to use and use the same for both places.

**Review by klausschuch — CHANGES_REQUESTED** [review]

File: schema/SystemStructureCommon.xsd (hunk 1 above)
Comment (pmai) [reply schema/SystemStructureCommon.xsd:491]: Added all accoringly.

**Comment by JochenKoehler** [comment]
Web meeting 2023-04-28

- [ ] Can be merged (by @pmai )

**Pull request merged**